.zenodo.json

```diff
@@ -65,18 +70,31 @@
             "orcid": "0000-0002-8546-9128",
             "affiliation": "Aperio Software Ltd."
         },
+        {
+            "name": "JunyanHuo"
```

Comment:
@PCJY Could you let us know your preferred name, affiliation and orcid if you have them? Thanks!

Reply:
```suggestion
            "name": "Junyan Huo",
            "orcid": "0000-0002-5469-7697",
            "affiliation": "University College London"
```